test/k8sT/manifests/iperf3-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: iperf3-server
+  labels:
+    id: iperf3-server
+    zgroup: testapp
+spec:
+  hostNetwork: true
+  terminationGracePeriodSeconds: 0
+  containers:
+  - name: iperf
+    image: docker.io/networkstatic/iperf3
```

Comment:
This image is a bit big (50MB) for what it is. Maybe we could build our own, or better, use netperf instead? We already have images with netperf used in e2e tests.

We also shouldn't be using the `:latest` tag.

Reply:
This netperf image does not support sctp. I dont know how it works to update that image in order to generate another one which supports sctp. What do you think?

Reply:
Ah, actually `docker.io/cilium/netperf:1.0` has iperf3.

Reply:
I have just tried, but the version of this iperf3 doesn't support the SCTP.  I believe we have to generate another image. Could you provide more information how I can go on with this?

Reply:
See https://hub.docker.com/r/cilium/netperf, you'll need to send a pull request to https://github.com/cilium/misc-scripts.

Reply:
@borkmann, @brb, regarding the comment of @pchaigno, I made this [commit](https://github.com/navarrothiago/misc-scripts/commit/2d4c6c43f226f9b2bdc7bbcbc5dbc5fe2c829c49) to change the image in order to support SCTP in iperf3. However, I believe this repository is not opened for PR. How we can proceed?

Reply:
> However, I believe the repository https://github.com/cilium/misc-scripts is not opened for PR.

What makes you say that?

Reply:
Oooh! Yes! My fault. Sorry about the confusion. I have already sent it.